internal/traceprof/traceproftest/overhead.txt

```diff
@@ -0,0 +1,43 @@
+name                                        old time/op           new time/op           delta
+EndpointsAndHotspots/direct/hello-world-16           6.52µs ± 1%           8.87µs ± 0%   +36.02%  (p=0.016 n=5+4)
+EndpointsAndHotspots/http/hello-world-16             83.8µs ± 0%           89.1µs ± 0%    +6.25%  (p=0.008 n=5+5)
+EndpointsAndHotspots/grpc/hello-world-16              104µs ± 0%            109µs ± 0%    +4.87%  (p=0.008 n=5+5)
+EndpointsAndHotspots/direct/cpu-bound-16              101ms ± 0%            101ms ± 0%      ~     (p=0.690 n=5+5)
+EndpointsAndHotspots/direct/io-bound-16               101ms ± 0%            101ms ± 0%      ~     (p=0.841 n=5+5)
+EndpointsAndHotspots/http/cpu-bound-16                101ms ± 0%            101ms ± 0%      ~     (p=0.841 n=5+5)
+EndpointsAndHotspots/http/io-bound-16                 101ms ± 0%            101ms ± 0%      ~     (p=0.095 n=5+5)
+EndpointsAndHotspots/grpc/cpu-bound-16                101ms ± 0%            101ms ± 0%      ~     (p=0.151 n=5+5)
+EndpointsAndHotspots/grpc/io-bound-16                 101ms ± 0%            101ms ± 0%      ~     (p=0.056 n=5+5)
```

Comment:
Interpretation: Latency overhead is only visible for the hello-world scenario (a request handler that does no work other than span creation). For anything approaching a realistic workload there is no overhead that is distinguishable from noise.